ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml

```diff
@@ -2455,17 +2455,7 @@ periodics:
           tests:
           - as: launch
             steps:
-              cluster_profile: hypershift
-              test:
-              - as: publish-console-url-hack
-                commands: echo "https://cluster.example.com" > "${SHARED_DIR}/console.url"
-                from: cli
-                resources: { requests: { cpu: 100m, memory: 100Mi } }
-              - as: wait
-                commands: sleep $(CLUSTER_DURATION) & wait
-                from: cli
-                resources: { requests: { cpu: 100m, memory: 100Mi } }
-              workflow: hypershift-launch-aws
```

Comment:
I know that `launch hypershift` doesnt necessarily work, but this change will completely break it.  The `ci-operator` requires a `cluster_profile` to launch it's clusters.

Reply:
@bradmwilliams I updated to keep the cluster_profile. FWIW I'm intending for basic launch hypershift to work after this PR and then follow up with improvements. Please let me know if there's anything else you think it would break.